README.md

```diff
@@ -1,45 +1,94 @@
-### Help
-In the event that links to documentation are broke:
-- If the link is to internal documentation in a files directory, it could be
-broken because the file was removed, renamed, or the repository path has been
-altered. If the document is still contained in the files directory, fix the link
-and push the change. If the document has been removed, remove the link.
-- If the link is to external documentation, check the [internet archive](https://archive.org/).
-If the documentation is on the archive, link to the archive and push the change.
-Otherwise remove the link.
+# Getting Started
+We are a university design team that represents [Queen's Unversity](https://www.queensu.ca/)
+in the [Canadian Satellite Design Team](https://www.csdcms.ca/) (CSDC). Any
+student (undergraduate and graduate) of Queen's or RMC is welcome to join the
+team.
+
+Getting started on a design team can be intimidating. Some common fears are:
+  * You are worried about making a mistake.
+  * You feel you lack the necessary or relevant technical knowledge.
+  * You are worried about over commiting and becoming overwhelmed during the
+     semester.
+
+We understand, we had the same fears and aprehensions when we first got involved.
+Let's talk about each one.
+  * We like mistakes. We have made some pretty big ones ourselves, including
+    frying a $6000 transceiver board.
+  * We'd like to help you learn. No previous space related or serious technical
+    knowledge is required.
+  * By using Github, all of our work is nicely synchronised and it is easy to
+    offload tasks. If you are working on something and have to stop, push your
+    most recent work and notify the relevant team lead you need to offload the
+    task.
+
+Alot of these fears come from thinking that our design team is going to be like
+working for a company. It isn't. It is more like an
+[open source](https://en.wikipedia.org/wiki/Open_source) project. Our team has
+a clear purpose:
+  > Launch a satellite designed, fabricated and validated by undergraduate/graduate students into space.
+
+If this excites you and you are willing to learn, you are a great fit for the
+team.  When determinng what you want to work on, the first choice is:
+- Do I want to program?
+- Do I want to design circuit boards?
+- Do I want to do CAD?
+
+After you decide that you can take a look through the [repositories](https://github.com/queens-satellite-team)
+and associated issues ([here](https://github.com/queens-satellite-team/moc-sat/issues) are the MocSat issues).
+Once you have found a project or issue that interests you:
+  - If you know how to fix the issue, do so and make a pull request to be
+    reviewed by the team lead.
+  - Post in the [new members](https://github.com/orgs/queens-satellite-team/teams/new-members)
+    team discussion with what your technical interest is (code, PCB design, CAD)
+    and sub-system. The team lead will then be able to give you more information.
+
```

Comment:
1.  I find this to be out of place in terms of the flow of the document.  I would add it at the end.  I think this would make more sense as here you address what people can do if they want to join the team and personally I would want to read all the information before making that decision.

2. In the situation where this document is a persons first introduction to the team.  I think there could be more information added for someone who has no idea how to make a pull request to fix a current issue.  The content seems contradictory of what you previously wrote. You said not to fear the lack the necessary or relevant technical knowledge and this may cause fear.